macosx/kext/AudioDevice.cpp

```diff
@@ -539,6 +559,53 @@ int kXAudioDevice::create_audio_controls(IOAudioEngine *audioEngine)
     audioEngine->addDefaultAudioControl(control);
     control->release();
     }
+    
+    {   // input control
+        
+        IOAudioLevelControl *control=NULL;
+
+        for(int i = 0; i < 8; i++)
+        {
+            const char *names[]=
+            {
+                kIOAudioControlChannelNameAll,
+                kIOAudioControlChannelNameLeft,
```

Comment:
kIOAudioControlChannelNameRight is missing, kIOAudioControlChannelNameAll is not required.